cli/ops/tty.rs

```diff
@@ -250,3 +254,70 @@ pub fn op_isatty(
     })?;
   Ok(JsonOp::Sync(json!(isatty)))
 }
+
+#[cfg(unix)]
+use libc::{c_int, c_ulong, winsize, TIOCGWINSZ};
+#[cfg(unix)]
+use std::mem::MaybeUninit;
+
+#[cfg(unix)]
+extern "C" {
+  fn ioctl(fd: c_int, request: c_ulong, ...) -> c_int;
+}
```

Comment:
Is this `extern "C"` necessary? Why can't you use the one in libc? 
https://docs.rs/libc/0.2.71/libc/fn.ioctl.html